docker/conf.d/php.ini

```diff
@@ -1,7 +1,7 @@
 [PHP]
 memory_limit = 256M
 error_reporting = E_ALL
-display_errors = Off
-display_startup_errors = Off
+display_errors = On
+display_startup_errors = On
```

Comment:
This isn't production, we should show anything that goes wrong. Only contributing developers are going to be running `build.sh`